bedrock/stories/templates/stories/stories-base-article.html

```diff
@@ -0,0 +1,136 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol.html" %}
+{% from "stories/includes/macros.html" import stories_card with context %}
+{% from "stories/includes/macros.html" import stories_image, stories_article_blockquote, stories_card with context %}
+{% from "macros.html" import image with context %}
+
+{% block page_css %}
+  {{ css_bundle('mozilla-stories-article')}}
+  {{ css_bundle('mozilla-stories-shared')}}
+  {{ css_bundle("protocol-card")}}
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle("stories-article")}}
+{% endblock %}
+
+{% block article_theme %}pink{% endblock %}
+
+{% block body_class %}theme-{{ self.article_theme() }}{% endblock %}
+
+{% block site_header %}
+  {# Bug 1438302 Avoid duplicate content for en-CA and en-GB pages. #}
+  {% if LANG == 'en-CA' %}
+    {% set logo_alt = ftl('navigation-v2-mozilla') + ' (CA)' %}
+  {% elif LANG == 'en-GB' %}
+    {% set logo_alt = ftl('navigation-v2-mozilla') + ' (UK)' %}
+  {% else %}
+    {% set logo_alt = ftl('navigation-v2-mozilla') %}
+  {% endif %}
+
+  <div class="c-navigation top-header-navigation mzp-is-sticky">
+    <div class="c-navigation-l-content">
+      <div class="c-navigation-container">
+        <button class="c-navigation-menu-button" type="button" aria-controls="c-navigation-items">{{ ftl('navigation-v2-menu') }}</button>
+        <div class="c-navigation-logo">
+          <a href="{{ url('mozorg.home') }}" data-link-name="mozilla home icon" data-link-type="nav">
+            <img class="c-navigation-logo-image" src="{{ static('protocol/img/logos/mozilla/logo-word-hor.svg') }}" alt="{{ logo_alt }}" width="112" height="32">
+          </a>
+        </div>
+        <div class="c-navigation-items" id="c-navigation-items">
+          {% include 'includes/protocol/navigation/nav-cta.html' %}
+          <div class="c-navigation-menu">
+            <nav class="c-menu mzp-is-basic">
+              <ul class="c-menu-category-list">
+                {% include 'includes/protocol/navigation/menus/firefox.html' %}
+                {% include 'includes/protocol/navigation/menus/products.html' %}
+                {% include 'includes/protocol/navigation/menus/whoweare.html' %}
+                {% include 'includes/protocol/navigation/menus/innovation.html' %}
+              </ul>
+            </nav>
+          </div><!-- close .c-navigation-menu -->
+        </div><!-- close .c-navigation-items -->
+      </div><!-- close .c-navigation-container -->
+    </div><!-- close .c-navigation-l-content -->
+    <nav aria-label="breadcrumb" class="mzp-c-breadcrumb mzp-is-sticky">
+      <ol class="mzp-c-breadcrumb-list">
+        <li class="mzp-c-breadcrumb-item"><a href="{{ url('stories.landing') }}">Stories</a></li>
```

Comment:
**issue (accessibility):** getting some contrast errors on the white text/pink background breadcrumb
<img width="668" alt="Screen Shot 2022-07-28 at 1 05 14 PM" src="https://user-images.githubusercontent.com/19650432/181500854-4073aea9-c860-433a-b95a-f08de1ef6424.png">

**suggestion:** this would be such a cool use case for the new[ color-contrast() function](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/color-contrast), but it's not supported yet. It's tricky because these backgrounds will change, but maybe keeping the black text of the heading is OK? likely need to check in with designer